bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/robot/ProblemRobot.scala

```diff
@@ -612,12 +612,16 @@ class ProblemRobot(s: ShellAdvisor) extends Robot {
       }
 
     private val noPIPhoneNumber = when (p.investigators.pi.phone.isEmpty) {
-      new Problem(Severity.Warning, s"No phone number given for ${investigatorFullName(p.investigators.pi, "PI")}. This is for improved user support.",
-        "Overview", s.inOverview{
-          _.editPi(_.Phone.requestFocus)
-        })
+      new Problem(Severity.Warning,
+        s"No phone number given for ${investigatorFullName(p.investigators.pi, "PI")}. This is for improved user support.",
+        "Overview", s.inOverview(_.editPi(_.Phone.requestFocus)))
     }
 
+    private val invalidPIPhoneNumber = (for {
+      badPhone <- p.investigators.pi.phone if badPhone.count(_.isDigit) < 10
```

Comment:
I feel very hesitant to do something like that. Extensions, for example, contain the letter x, typically, and although I don't think there are any other cases in the institutions list now of anything else, it is entirely possible that something might come up. Additionally, how complicated would we want to take this? Parentheses (in which case, do we only allow one set, require them to be balanced, etc), a multitude of separators [.-/], and all kinds of things could crop up, as could an optional leading +, as you mention.

I was trying to find a regex online for phone numbers as a curiosity, and even one for just North America was ridiculously complicated. I don't think we want to take things that far, at least until science requests it.